transports/noise/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 - Update to `libp2p-core` `v0.35.0`.
 
+- Migrate testing from against sodiumoxide to libsodium-sys-stable and ed25519-compact
```

Comment:
I don't think this is relevant for a user, thus I suggest not mentioning this in the changelog.

Reply:
Users are developers and I as user of this library would like to know what crypto impl(s) is / are being used for what.

Reply:
I don't think the majority of folks cares what libraries are being used for testing.

Reply:
Up to you - was a worth a try :) I'll change it

Reply:
Done - Reverted that commit.